### PR TITLE
Cli version check

### LINKED ISSUE #608: Request for a couple of options on command line
Could you add in a couple options on the pihole command?

This one should be pretty easy.
-v:   Display pihole version and web version

-(letter of your choice):  Query for any available updates to pihole.  Could possibly combine this check into the -v check above.  Show what's running and show what's available.

Edit (Mcat12): Looks like the `<` and `>` from `letter of your choice` made the whole thing invisible. Changed to parenthesis.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/36564548-request-for-a-couple-of-options-on-command-line?utm_campaign=plugin&utm_content=tracker%2F3011939&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F3011939&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by PromoFaux**
Fixes #608

Changes proposed in this pull request:

- add command `pihole -v` to display installed versions and latest versions

```bash
::: Pi-hole version is v2.8.1 (Latest version is v2.8.1)
::: Web-Admin version is v1.3 (Latest version is v1.3)
```
 

@pi-hole/gravity

### CODE REVIEW

File: pihole [hunk 1]
```diff
@@ -90,6 +95,7 @@ function helpFunc {
 	echo ":::  -s, setupLCD				Automatically configures the Pi to use the 2.8 LCD screen to display stats on it"
 	echo ":::  -c, chronometer			Calculates stats and displays to an LCD"
 	echo ":::  -h, help				Show this help dialog"
+	echo ":::  -v, version          Show current versions"
```
Comment (Mcat12) [inline pihole]: I think the spacing is off. (though it might not look different in the terminal unless tabs are different than 4 spaces)

**Comment by brantje** [comment]
Approved, looks good

**Comment by PromoFaux** [comment]
Need to fix that spacing though :S

**Comment by PromoFaux** [comment]
Got there eventually!
```bash
pi@raspberrypi:/etc/.pihole $ pihole
::: Control all PiHole specific functions!
:::
::: Usage: pihole [options]
:::             Add -h after -w (whitelist), -b (blacklist), or -c (chronometer)  for more information on usage
:::
::: Options:
:::  -w, whitelist            Whitelist domains
:::  -b, blacklist            Blacklist domains
:::  -d, debug                Start a debugging session if having trouble
:::  -f, flush                Flush the pihole.log file
:::  -u, updateDashboard      Update the web dashboard manually
:::  -g, updateGravity        Update the list of ad-serving domains
:::  -s, setupLCD             Automatically configures the Pi to use the 2.8 LCD screen to display stats on it
:::  -c, chronometer          Calculates stats and displays to an LCD
:::  -h, help                 Show this help dialog
:::  -v, version              Show current versions
:::  uninstall                Uninstall Pi-Hole from your system :(!

```

**Comment by PromoFaux** [comment]
approved

**Comment by jacobsalmela** [comment]
Approved

Works as expected.

**Pull request merged**